akka-cluster-sharding/src/main/scala/akka/cluster/sharding/protobuf/ClusterShardingMessageSerializer.scala

```diff
@@ -266,6 +277,14 @@ private[akka] class ClusterShardingMessageSerializer(val system: ExtendedActorSy
     ShardStats(parsed.getShard, parsed.getEntityCount)
   }
 
+  private def startEntityToByteArray(s: StartEntity): Array[Byte] = {
+    s.entity.getBytes(StandardCharsets.UTF_8)
+  }
+
+  private def startEntityFromBinary(bytes: Array[Byte]): StartEntity = {
+    StartEntity(new String(bytes, StandardCharsets.UTF_8))
+  }
```

Comment:
this is nice and simple now but if we want to evolve this message for some reason we don't have it in protobuf, I think we should protobuf it right away